classes/requests/helpers/class-ledyer-woocommerce-bridge.php

```diff
@@ -84,7 +84,8 @@ private static function set_order_settings( $full = true ) {
 
 		self::$ledyer_settings = array(
 			'security' => array(
-				'level' => intval( ledyer()->get_setting('security_level') )
+				'level' => intval( ledyer()->get_setting('security_level') ),
+				'requireClientValidation'  => true,
```

Comment:
requireClientValidation is set in a conditional statement further below, but it should also be set here I guess, so it's set in both cases.